_episodes_rmd/08-making-packages-R.Rmd

```diff
@@ -147,6 +147,8 @@ Overall, your package directory should look something like this:
 
 <img src="../fig/R-package-structure.svg" alt="R Package Structure" width="500" />
 
+In the diagram above, your package is a folder (the same as it would exist on your computer!). This folder contains several files and sub-folders. For example, the folder 'R' contains all of the R scripts you just wrote above.
+
```

Comment:
```suggestion
```

Please note: I am resolving old issues ahead of the transition to the workbench (as pull requests will need to be resolved by then). My aim is to address any remaining issues that the maintainers were looking for.  

Since alt text for this was added in https://github.com/swcarpentry/r-novice-inflammation/pull/510, we no longer need this description attatched to the image.